app/models/manageiq/providers/base_manager/metrics_capture.rb

```diff
@@ -6,4 +6,152 @@ def initialize(target, ems = nil)
     @target = target
     @ems = ems
   end
+
+  def zone
+    ems.zone
```

Comment:
How will this work if `ems` isn't passed up [here](https://github.com/ManageIQ/manageiq/pull/19541/files#diff-62cd5f478c94c6f643d7a8f6d8262251L7) ?
Should we maybe try `@ems = ems || target.ext_management_system` and raise if `@ems` is nil there?

Reply:
I had that in the code, but pulled out because I didn't want an extra ems lookup.
Since I think we are passing in zone all the time, seems like a no op (but wasn't sure if we wanted the extra complexity)

Should we put back in?

I only found 1 place where we reference `MetricsCapture`:

- [capture.rb](https://github.com/ManageIQ/manageiq/blob/master/app/models/metric/ci_mixin/capture.rb#L4)

Reply:
Okay, maybe we shouldn't have it be an optional parameter if we're going to rely on it being set?